src/controllers/signup.controller.js

```diff
@@ -0,0 +1,40 @@
+import SignupService from '../services/signup.service';
+import BcryptService from '../services/bcrypt.service';
+import Response from '../helpers/response';
+
+const response = new Response();
+/**
```

Comment:
Can you use a static method so that there is no need to instantiate the object every time?